test/e2e/antctl_test.go

```diff
@@ -33,6 +33,32 @@ type cmdAndReturnCode struct {
 	expectedReturnCode int
 }
 
+// TestAntctl is the top-level test which contains all subtests for
+// Antctl related test cases as they can share setup, teardown.
+func TestAntctl(t *testing.T) {
+	skipIfHasWindowsNodes(t)
+	skipIfNotRequired(t, "mode-irrelevant-case")
```

Comment:
```suggestion
	skipIfNotRequired(t, "mode-irrelevant")
```
otherwise all keywords need a redundant suffix to keep consitency.

Reply:
done